bin/i18n/codeorg_crowdin.yml

```diff
@@ -1,7 +1,7 @@
 #
 # Your crowdin's credentials
 #
-"project_identifier" : "codeorg"
+"project_id" : "26074"
```

Comment:
Changing the project_id might impact this code in the current version of the sync-down https://github.com/code-dot-org/code-dot-org/blob/85647ca40950ed2493cc3768a3173160b9bc74b7/bin/i18n/sync-down.rb#L29 . Just want to put it under your radar.

Reply:
good catch!